src/pages/Staking/StakesList/DelegateStake/DelegateStake.component.tsx

```diff
@@ -29,9 +24,9 @@ export const DelegateStakeComponent = ({
   currentDelegate,
   onCancelDelegation,
 }: DelegateStakeComponentProps) => {
-  const { control, formState, handleSubmit } = useForm<DelegateStakeValues>({
+  const { control, formState, handleSubmit } = useForm<DelegateValues>({
```

Comment:
It's not a bad idea to extract this type however I would prefer to keep seperate type for each form to have the consistent structure and make it easy to maintain. What do you think about keeping the `DelegateStake.fields` file and export values this way:
``` typescript
export type DelegateStakeValues = DelegateValues
export ...
```

Reply:
It shouldn't break your DelegateFeeEstimator since `DelegateStakeValues` will be extending `DelegateValues`